src/test/java/seedu/anilist/model/genre/GenreTest.java

```diff
@@ -33,25 +39,24 @@ public void isValidGenreName() {
         assertThrows(NullPointerException.class, () -> Genre.isValidGenreName(null));
 
         // invalid genres
-        // genre containing non-alphanumerical characters
-        assertFalse(Genre.isValidGenreName(INVALID_GENRE_NON_ALPHANUMERICAL));
-        // genre starting with space
-        assertFalse(Genre.isValidGenreName(INVALID_GENRE_START_WITH_SPACE));
-        // genre with double spaces
-        assertFalse(Genre.isValidGenreName(INVALID_GENRE_WITH_DOUBLE_SPACE));
+        assertFalse(Genre.isValidGenreName(INVALID_STRING_SPACE));
+        assertFalse(Genre.isValidGenreName(INVALID_STRING_EMPTY));
+        assertFalse(Genre.isValidGenreName(INVALID_STRING_NON_ASCII));
+        assertFalse(Genre.isValidGenreName(INVALID_GENRE_ALPHA));
+        assertFalse(Genre.isValidGenreName(INVALID_GENRE_NUMERIC));
+        assertFalse(Genre.isValidGenreName(INVALID_GENRE_NON_ALPHANUMERIC));
 
         // valid genres
-        // genre with lower case
-        assertTrue(Genre.isValidGenreName(VALID_GENRE_LOWER_CASE));
-        // genre with mixed case
-        assertTrue(Genre.isValidGenreName(VALID_GENRE_MIXED_CASE));
+        assertTrue(Genre.isValidGenreName(VALID_GENRE_ACTION));
+        assertTrue(Genre.isValidGenreName(VALID_GENRE_SCIENCE_FICTION));
+        assertTrue(Genre.isValidGenreName(VALID_GENRE_SUPERNATURAL));
```

Comment:
doesn't really show the equivalence partitioning here anymore, but I guess it is inside the commandTestUtil instead?

Reply:
might want to add comments here anyways to show the equivalence partitioning

Reply:
added